feature/bgp/linkbandwidth_community/otg_tests/linkbandwidth_aggregation/README.md

```diff
@@ -0,0 +1,83 @@
+# RT-7.6: BGP Link Bandwidth Community - Aggregation
+
+## Summary
+
+This test verifies Link-bandwidth (LBW) extended community aggregation feature by DUT.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+    ```
+                                          |         |
+    [ 64x eBGP ] --- [ ATE Port 1 ] ----  |   DUT   | ---- [ ATE Port 2 ]
+                                          |         |
+    ```
+
+### RT-7.6.1: Verify LBW aggregation to eBGP peer
+
+* Configure DUT with routed ports on DUT.
+* Configure 64x eBGP peers on ATE Port 1 interface in peering group UPSTREAM.
+* Configure a single eBGP peer on ATE Port 2 interface in peering group DOWNSTREAM.
+* Enable BGP LBW receive for peering group UPSTREAM.
+* Enable BGP LBW send for peering group DOWNSTREAM. Enable Link Bandwidth aggregation feature.
+
+**TODO:** Link Bandwidth aggregation feature is not currently modeled in OC.
```

Comment:
Is the intent to support [draft-ietf-bess-ebgp-dmz](https://datatracker.ietf.org/doc/draft-ietf-bess-ebgp-dmz/)?  Please add a reference link as it may help with clarity on the intent of the test.

I have opened a OpenConfig model update here for these paths.  Please reivew:
https://github.com/openconfig/public/pull/1131